src/metabase/email/creator_sentiment_email.mustache

```diff
@@ -0,0 +1,36 @@
+{{> metabase/email/_header }}
+  <div class="card-container">
+    <p style="font-weight: 500; font-size: 0.875em; line-height: 1.375em; color: {{colorTextMedium}};">
```

Comment:
This paragraph shouldn't be shown if the instance has the `hosting` flag _enabled_.
Can't say if you included that in the clojure code, but in any case the `<p>` shouldn't be there, even if empty, in those cases.